data_collection/web/guides.rst

```diff
@@ -739,23 +800,30 @@ For fully static pages calling :ref:`enableLinkTracking<jtc-api-enableLinkTracki
 Goal tracking
 -------------
 
-At this point we have tracked lots of various typose of events. We have regular page views, we have downloads, outlinks, custom events and others. Above them all there's one more event type we can track: a conversion.
-And goal tracking is about tracking conversions. If you can point out parts of your website/application more important from your bisness perspective, you could :ref:`define those parts as goals<https://help.piwik.pro/support/analytics-new/goals/>`.
+At this point we have tracked many different types of events. We have regular page views, downloads, outlinks, custom events and others. Above them all there's one more event type we can track: a conversion.
+And goal tracking is about tracking conversions. If you can point out parts of your website/application more important from your business perspective, you could :ref:`define those parts as goals<https://help.piwik.pro/support/analytics-new/goals/>`.
 Visiting a specific landing page, submitting a contact form, downloading a PDF file with your product manual - these are popular examples of goal definitions. You can even define a goal based on the custom event you are tracking.
 
-Once the goal is defined, every time a tracked event fits its definition, an additional conversion event will be created. We call this procedure an "automatic conversion".
+If a goal with automatic tracking is defined in Analytics, every time an events matching the goal's definition is tracked, we create an additional conversion event and save it along the original event. We call this procedure an "automatic conversion".
 
-Alternatively, you can trigger a goal manually with the used of
+.. note::
+
+    `Set up goals <https://help.piwik.pro/support/analytics-new/goals/>`_ article shows how to define a goal triggered by visiting a specific page.
+
+Alternatively, you can trigger a goal manually with the use of :ref:`trackGoal<jtc-api-trackGoal>` function
 
 .. code-block:: javascript
 
     // force conversion of the goal with ID 17
     _paq.push(["trackGoal", 17]);
 
-before `trackPageView` was triggered.
+before triggering :ref:`trackPageView<jtc-api-trackPageView>`.
+
+We call this procedure a "manual conversion". Manual conversion doesn't create an additional conversion event like the automatic conversion does.
```

Comment:
```suggestion
We call this procedure a "manual conversion". Manual conversion will sent a new standalone event immediately and is not tied to any other events sent by tracker like automatic conversions.
```
I tried to say that goal event is sent immediately when this command is processed (without waiting on page view).